cmd/thanos/query.go

```diff
@@ -81,6 +82,8 @@ func registerQuery(app *extkingpin.App) {
 
 	instantDefaultMaxSourceResolution := modelDuration(cmd.Flag("query.instant.default.max_source_resolution", "default value for max_source_resolution for instant queries. If not set, defaults to 0s only taking raw resolution into account. 1h can be a good value if you use instant queries over time ranges that incorporate times outside of your raw-retention.").Default("0s").Hidden())
 
+	defaultLabelLookbackDelta := cmd.Flag("query.labels.api-lookback-delta", "The default lookback duration for retrieving labels through Labels API when the range parameters are not specified. The zero value means range covers the time since the beginning.").Default("0s").Duration()
```

Comment:
Ok, I would not overuse `lookback-delta` naming. This is really specific to PromQL and it means quite a specific thing -> not queried time range, but rather the time when the series is considered dead. Maybe just `query.metadata.default-time-range`?

Reply:
I propose `metadata` as I would love to cover `series` as well (on top of `labels, label_value`) I think it makes sense to group together, no need for flag for each of those :thinking: WDYT?